crates/common/mqtt_channel/src/config.rs

```diff
@@ -376,3 +404,22 @@ impl Config {
         Ok(mqtt_options)
     }
 }
+
+/// Read the first line of the given file and return it.
+pub fn read_password(path: impl AsRef<Path>) -> Result<String, CertificateError> {
```

Comment:
To have a consistent use of zeroize.


```suggestion
pub fn read_password(path: impl AsRef<Path>) -> Result<Zeroize<String>, CertificateError> {
```